tests/test_inputs.py

```diff
@@ -183,3 +272,34 @@ def test_build_from_sample(self):
 
         # This should verify we don't have any issues in the object created for display from a sample
         self.assertEqual(type(disp_datatable), dash_table.DataTable)
+
+    def test_build_from_state(self):
+        employees = ["A-Mgr", "B-Mgr", "C", "D", "E", "E-Tr"]
+        shifts = [str(i+1) for i in range(14)]
+
+        # Make every employee available for every shift
+        availability = {
+            "A-Mgr": [1] * 14,
+            "B-Mgr": [1] * 14,
+            "C": [1] * 14,
+            "D": [1] * 14,
+            "E": [1] * 14,
+            "E-Tr": [1] * 14,
+        }
+
+        state = asarray([
+            # Give managers alternating shifts
+            [i % 2 for i in range(14)],
+            [(i+1) % 2 for i in range(14)],
+            [1 for _ in range(14)],
+            [1 for _ in range(14)],
+            [1 for _ in range(14)],
+            [1 for _ in range(14)],
+        ])
```

Comment:
@k8culver noticed that the availability for this test specifies 14 shifts, but the state originally only described 5 shifts. I fixed it here, but this error is also present in `test_build_from_sample`. Can fix it there as well or wait for your PR to be merged. Also could be a question of the robustness of this specific test if this mismatch is allowed to occur without errors.

Reply:
I think I initially broke the tests with my PR implementing 14 week schedules because I forgot to update the tests and then I think Theodor updated them to fix my forgetfulness in this [PR](https://github.com/dwave-examples/employee-scheduling/commit/ca438938f25e6653ff13fcef697422f59688d4b3). So I think it's just a mistake (but that's a good point that it's a bit concerning that it works with that mismatch). Feel free to make any changes to make the tests make more sense or if you want me too I can instead.